src/martok/Martok.ts

```diff
@@ -106,6 +111,10 @@ export class Martok {
     this.externalSymbols.length = 0;
   }
 
+  public clearAdditionalDeclarations() {
+    this.additionalDeclarations.length = 0;
```

Comment:
whoa, typescript is weird. surprised this clears the array instead of calling `clear()` or setting `additionalDeclarations` to an empty array.